src/main/java/com/tobiasdiez/easybind/EasyObservableList.java

```diff
@@ -68,6 +69,19 @@ default BooleanBinding isEmptyBinding() {
         filteredList.predicateProperty().bind(predicate);
         return filteredList;
     }
+    
+	/*
+	 * Sadly, we cannot easily let {@link FilteredList} implement {@link EasyObservableList}, because that class is final. Neither can we return
+	 * both at once, as intersection types do not exist in Java. Therefore, the easiest way is to simply create two methods, one for each return
```

Comment:
> be fine if we copy-paste the FilteredList class into EasyBind

Except that this doesn't work because `FilteredList` references some private `com.sun.javafx` classes :/

Reply:
😞 ok, then we keep your solution using `filteredWrapped` for now, and change it as soon as `FilteredList` is no longer final in the jdk.